src/jaystring.js

```diff
@@ -26,7 +28,10 @@ function jaystring(item) {
     throw new Error('Can stringify only either normal or arrow functions')
   } else if (typeof item === 'object') {
     const proto = Object.getPrototypeOf(item)
-    if (item instanceof RegExp && proto === RegExp.prototype) return String(item)
+    if (item instanceof RegExp && proto === RegExp.prototype) {
+      // String(regex) is not ok on Node.js 10 and below: console.log(String(new RegExp('\n')))
+      return `new RegExp(${jsval(item.source)}, ${jsval(item.flags)})`
```

Comment:
Instead of exporting `jsval`, why don't we use `format`?
```suggestion
      return format('new RegExp(%j, %j)', item.source, item.flags)
```

Reply:
That changes the return type and makes it inconsistent between the two branches.
But we don't re-export this lib, so that should be fine too.

Reply:
Actually, let's do this.